src/components/Camera/index.tsx

```diff
@@ -53,16 +58,29 @@ const Camera: FunctionComponent<Props> = ({
   translate,
   webcamRef,
 }) => {
+  // Specify just a camera width (no height) because on safari if you specify both
+  // height and width you will hit an OverconstrainedError if the camera does not
+  // support the precise resolution.
+  // Using width here because on some special devices (e.g. Samsung Galaxy),
+  // setting 720px in height results to 720x960 resolution instead of the desired 720x1280.
+  //
+  // Resolution needs to be set to a lower value if WebM format is not supported
+  // as video formats like MP4 (Safari) result in larger video file sizes
+  // * Resolutions: 1280px = 720p, 480px = VGA (VGA is minimum we can go for automation + iOS is using VGA resolution)
+  const defaultCameraWidthInPx = isWebmFormatSupported() ? 1280 : 480
```

Comment:
Is `480` too low? Can we use something higher (like `560` or `720`)?

Reply:
Discussed with Bio team & it seems that iOS SDK was using way lower resolution (VGA) setting than Web. But I'll do some tests to see how big the file sizes are at `560` in case that's viable. I tried `720` and that did still result in some of the over 10s videos being over 10MB in size.